flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/testutils/MockRecordEvaluator.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.flink.streaming.connectors.kafka.testutils;
+
+import org.apache.flink.connector.base.source.reader.RecordEvaluator;
+import org.apache.flink.table.data.RowData;
+
+/** A mock record evaluator. */
+public class MockRecordEvaluator implements RecordEvaluator<RowData> {
```

Comment:
Would it be simpler to define it as a public class in `KafkaTableITCase`, similar to the existing public class `KafkaTableITCase#TestPartitioner`?

Reply:
KafkaDynamicTableFactoryTest relies on it too.